exporter/datadogexporter/internal/metrics/series.go

```diff
@@ -67,6 +69,17 @@ func DefaultMetrics(exporterType string, hostname string, timestamp uint64, tags
 	return metrics
 }
 
+// GatewayUsageGauge creates a gauge metric to report is there is a gateway
+func GatewayUsageGauge(timestamp uint64, hostname string, tags []string, gatewayUsage *attributes.GatewayUsage) datadogV2.MetricSeries {
+	series := NewGauge("datadog.otel.gateway", timestamp, gatewayUsage.Gauge(), tags)
```

Comment:
What if the gateway host is from traces, metrics and logs, does this gauge report value of 3?

Reply:
`gatewayUsage.Gauge()` returns either `1` or `0` so in your case the value is `1`.